packages/mds-types/index.ts

```diff
@@ -340,7 +340,7 @@ export interface ComplianceResponse {
 // the geojson FeatureCollection type.
 export interface Geography {
   geography_id: UUID
-  geography_json: FeatureCollection
+  geography_json?: FeatureCollection
```

Comment:
FYI @drtyh2o @janedotx this caused `yarn start` to fail on mds-audit due to a return type mismatch on `mds-utils` `getPolygon`. Just curious.... shouldn't that be caught by the linter? And do all mds-core tests run whenever a PR is created/updated?

If we want to keep this change I made a little PR to fix it here: https://github.com/lacuna-tech/mds-core/pull/18